awpy/analytics/wpa.py

```diff
@@ -30,9 +31,57 @@ def round_win_probability(ct_score: int, t_score: int, map_name: str) -> dict:
     Args:
         ct_score (int): CT Score
         t_score (int): T Score
-        map_name (str): Map the demo is from
+        map_name (str): Map name the demo is from
 
     Returns:
-        A dictionary containing the CT game win, T game win and Draw probabilities
+        A dictionary containing the game win probability
     """
-    raise NotImplementedError
+    # Load the data from the json file
+    wpa_data = json.loads("../data/wpa/wpa.json")
+
```

Comment:
I think the `wpa_data should be loaded in the data `__init__` file and just imported in wpa.py.

Reply:
Good idea!